src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js

```diff
@@ -11,12 +12,16 @@ class GlobalTooltip extends Component {
     window.removeEventListener('mousemove', this.moveTooltip);
   }
 
-  moveTooltip = event => {
-    this.globalTooltip.style.left = `${event.pageX}px`;
-    this.globalTooltip.style.top = `${event.pageY -
-      this.globalTooltip.offsetHeight -
-      15}px`;
-  };
+  moveTooltip = _.throttle(event => {
+    const left = event.pageX;
+    const top = event.pageY;
+    setTimeout(() => {
```

Comment:
Fair. Realized we also don't need the height to calculate tooltip position, but that would be a bit more of a rewrite, so maybe in the future we can do that :)